src/index.html

```diff
@@ -12,4 +12,9 @@
 <body>
   <app-root>Loading...</app-root>
 </body>
+<script>
```

Comment:
Terrible way of adding active class to the element, will be fixed in original pull-request.